modules/misc/gnome_update_custom_keybindings.py

```diff
@@ -0,0 +1,109 @@
+# -*- coding: utf-8 -*-
```

Comment:
I think this file should be moved into a project of its own and packaged in Nixpkgs, then it would also be of more general use. I'm sure there would be plenty of non-HM users interested in this functionality.

I'd like to avoid having anything but small and simple Bash in HM. The only exception to this, so far, has been the `systemd-activate.rb` script and I have been on a multi-year quest to remove it. See #1618 for my latest attempt.